polyfill/intersection-observer-test.js

```diff
@@ -941,8 +941,9 @@ describe('IntersectionObserver', function() {
 
   });
 
-  describe('same-origin iframe', function() {
+  describe('iframe', function() {
     var iframe, win, doc;
```

Comment:
Didn't notice this before, but my editor is telling me `win` and `doc` are not used here.